src/pickup.cpp

```diff
@@ -171,6 +171,44 @@ static pickup_answer handle_problematic_pickup( const item &it, bool &offered_sw
     return static_cast<pickup_answer>( choice );
 }
 
+void Pickup::query_thief()
+{
+    player &u = g->u;
+    bool force_uc = get_option<bool>( "FORCE_CAPITAL_YN" );
+    const auto allow_key = [force_uc]( const input_event & evt ) {
+        return !force_uc || evt.type != CATA_INPUT_KEYBOARD ||
+               // std::lower is undefined outside unsigned char range
+               evt.get_first_input() < 'a' || evt.get_first_input() > 'z';
+    };
+    std::string answer = query_popup()
+                         .context( "YES_NO_ALWAYS_NEVER" )
+                         .message( "%s",
+                                   _( "Picking up this item will be considered stealing, continue?" ) )
+                         .option( "YES", allow_key ) // yes, steal all items in this location that is selected
+                         .option( "NO", allow_key ) // no, pick up only what is free
+                         .option( "ALWAYS", allow_key ) // Yes, steal all items and stop asking me this question
+                         .option( "NEVER", allow_key ) // no, only grab free item and never ask me again
+                         .cursor( 1 ) // default to the third option `QUIT`
```

Comment:
hey can you fix my copypasta here.  should ofc be `// default to the second option, "NO"`